e2e/benchmarks/local-benchmark/dump.js

```diff
@@ -0,0 +1,266 @@
+  const backends = Object.keys(jsonObjects);
+  if (Object.keys(jsonObjects[backends[0]]).length == 0) {
+    return;
+  }
+  for (let i = 0; i < backends.length; i++) {
+    const object = jsonObjects[backends[i]];
+    const fileName = `${newPrefix}${backends[i]}.json`;
+    const a = document.createElement('a');
+    const file = new Blob([JSON.stringify(object)], {type: 'application/json'});
+    a.href = URL.createObjectURL(file);
+    a.download = fileName;
+    a.click();
+    // This log informs tools file has been saved.
+    console.log(fileName);
+  }
+}
+
+async function convertTensorToData(tensor, needInfo = false) {
+  const data = await tensor.data();
+  const info = {value: data, shape: tensor.shape, dtype: tensor.dtype};
+  tensor.dispose();
+  if (needInfo) {
+    return info;
+  }
+  return data;
+}
+
+async function getPredictionData(output, needInfo = false) {
+  if (output instanceof Promise) {
+    output = await output;
+  }
+
+  if (output instanceof tf.Tensor) {
+    output = [await convertTensorToData(output, needInfo)];
+  } else if (Array.isArray(output)) {
+    for (let i = 0; i < output.length; i++) {
+      if (output[i] instanceof tf.Tensor) {
+        output[i] = await convertTensorToData(output[i], needInfo);
+      }
+    }
+    return output;
+  } else if (output != null && typeof output === 'object') {
+    for (const property in output) {
+      if (output[property] instanceof tf.Tensor) {
+        output[property] =
+            await convertTensorToData(output[property], needInfo);
+      }
+    }
+  }
+  return output;
+}
+
+
+/**
+ * Create a NamedTensorMap from an output node name.
+ * @param outputNodeName Output node name.
+ * @param modelJson The parsed model.json.
+ * @param dumpedJson The dumped tensor infomation (including shape, dtype,
+ *     value).
+ *
+ * @returns A NamedTensorMap.
+ */
+async function createNamedTensorMap(outputNodeName, modelJson, dumpedJson) {
+  const modelNodes = modelJson['modelTopology']['node'];
+  let inputs = [];
+  for (let i = 0; i < modelNodes.length; i++) {
+    if (outputNodeName === modelNodes[i].name && modelNodes[i].input) {
+      inputs = modelNodes[i].input;
+      break;
+    }
+  }
+  // In
+  // https://storage.googleapis.com/tfhub-tfjs-modules/mediapipe/tfjs-model/face_landmarks_detection/attention_mesh/1/model.json,
+  // some inputs are prefixed with '^'.
+  if (!inputs || inputs.length == 0 || inputs[0].startsWith('^')) {
+    return null;
+  }
+
+  let tensorMap = {};
+  for (let i = 0; i < inputs.length; i++) {
+    const key = inputs[i].split(':')[0];
+    if (dumpedJson[key] == null || dumpedJson[key][0] == null) {
+      console.warn('Tensor ' + key + ' is null!');
+      return null;
+    }
+    const tensorInfo = dumpedJson[key][0];
+    const tensor = tf.tensor(
+        Object.values(tensorInfo.value), tensorInfo.shape, tensorInfo.dtype);
+    tensorMap[key] = tensor;
+  }
+
+  return tensorMap;
+}
+
+async function predictAndGetData(
```

Comment:
This shouldn't be part of this file. If we don't need to dump, e2e should not have any dependency with this file. We need to split the predict and getData.

Reply:
Done, thanks!